### PR TITLE
T267293: font size feature fine tuning

### PR DESCRIPTION
**Pull request by medied**
https://phabricator.wikimedia.org/T267293

### Objective

As part of the font size feature [fine tuning](https://phabricator.wikimedia.org/T267293#6682271) and in lieu of the upcoming release, this is an option for what design wants prioritized for the release: having the `TextSize` component be rendered with no popup on the back. This PR aims to have both the CSK and the backspace key return back to article menu while keeping the article menu selected/highlighted state. ~I don't particularly like to query the `document` like that, but wanted to present this solution either way given the release timeline~ _revisited with further commits (#297)_

### Note

Follow up for #290

### CODE REVIEW

**Review by hueitan — COMMENTED** [review]
very bad idea, but i know this is just a temp fix for the release.

**Comment by medied** [comment]
Actually realized we can achieve the same effect with `containerRef`, so moving away from querying `document`

**Review by hueitan — APPROVED** [review]
looks good

File: style/popup.less [hunk 1]
```diff
@@ -24,5 +24,9 @@
 			max-height: unset;
 			top: 0;
 		}
+
+		&.hidden {
```
Comment (stephanebisson) [inline style/popup.less:28]: What is this for?

Comment (medied) [reply style/popup.less:28]: Oh good catch, that's leftover from the previous approach so removing...

**Pull request merged**